### ISSUE TITLE
Position marking

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#392]
majkinetor on https://lobste.rs/s/q02egw/10_days_left_for_next_browser suggests we added a famous Vimperator / Vim feature, that of marking positions.  It's not hard to do, we could even name the marks, persist them, fuzzy-search them, etc.

**Comment by jmercouris** [comment]
A great idea! It will also be a necessary component in adding inline notes on web pages and storing/reinflating the positions of notes in the future!

**Comment by Ambrevar** [comment]
Fixed with https://github.com/atlas-engineer/nyxt/commit/c9398328886031449c6f9d79c9e5b7e8c2524c68.

**Comment by Ambrevar** [comment]
Oops, wrong issue, reopening.

**Comment by pdelfino** [comment]
Hi,

Nyxt has a fetaure called `reading-lines` described [here](https://nyxt.atlas.engineer/article/reading-line.org). This feature was mentioned/release in a about a year in the middle of 2020. This is about 1 year **after** this issue was opened.

Is this issue still relevant? Did I miss something about the feature request?

**Comment by Ambrevar** [comment]
Indeed, this is unrelated to `reading-lines`.  Position marking is about adding a virtual "mark" to your page scroll position.

Say you scroll to 50% of the page and add a mark named `a`.  Then scroll further.  Now you want to go back to `a`, you can just call a command which would prompt for the list of available marks in the page and bring you there.
